sass/layout/_base.scss

```diff
@@ -0,0 +1,87 @@
+* {
+    box-sizing: border-box;
+}
+body {
+    background: #f7f7f7;
+    font-family: "Source Sans Pro", sans-serif;
+    margin: 0;
+    color: #6F7A8B;
+}
+
+ul {
+    margin-top: 0;
+    margin-bottom: 10px;
+}
+
+#main-header {
+    background: white;
+    box-shadow: 0 2px 2px rgba(0,0,0,.05),0 1px 0 rgba(0,0,0,.05);
+    left: 0;
+    min-height: 60px;
+    right: 0;
+    position: fixed;
+    top: 0;
+    z-index: 1030;
+
+    .navbar-header {
+        background: #67cea6;
+        color: white;
+        width: 200px;
+        line-height: 60px;
+        padding: 0 20px;
+        font-size: 20px;
+    }
+}
+
+#main-sidebar {
+    height: 100%;
+    margin: 0;
+    padding: 60px 0 0;
+    left: 0;
+    position: fixed;
+    padding-bottom: 0;
+    width: 200px;
+    background: #2a2d33;
+    color: #8a919f;
+
+    ul.navigation {
+        padding: 10px 0 0;
+        li {
+            position: relative;
+            a {
```

Comment:
When we start working on the actual UI, let's try not to nest further than three levels. Makes it harder to keep track of what's going on and styles become too specific. Also, it slows down rendering times having to parse through all of the selectors.

Using BEM should help alleviate this.